src/documents/components/FileDetail/__tests__/EditExistingFile.test.js

```diff
@@ -139,4 +139,4 @@ it('edits an existing file correctly', async () => {
   });
   await wait();
   expect(tree.container).toMatchSnapshot();
-}, 8000);
+}, 10000);
```

Comment:
if you run your local test using `yarn test -u -i` it runs test in a series and helps prevent some of the random timeout errors. This way we don't have to just randomly adjusts these timeout values